### DISCUSSION TITLE
Running an executable file still runs the latest version of the game

### DISCUSSION THREAD

**Discussion by DXB6** [SteamRE/DepotDownloader#181] [Q&A]
Now I just installed an older version of Subnautica, specificly the last one from 2015, and I am having an issue with nothing other than running the executable file. Each time I run the .exe inside the depot folder, it runs the games config perfectly fine, then I run the game, and it emmediatly closes it and re-opens it via steam with the newest version of the game. I have looked at the text document within the game folder, and it's date is in fact 12/23/2015; it just doesnt work how you would expect. I don't know if this issue has been reported multiple times with multiple games or not, or if its just this one game I'm having trouble with -- but if I could get some help on this issue, that would be fantastic as I have been working away at figuring out depotdownloader for a while today, and I've been really wanting to play old versions of games using this application.

**Comment by psychonic** [comment]
It sounds like DepotDownloader is working fine in this case, and the app just makes use of SteamAPI_RestartAppIfNecessary.

**Comment by DXB6** [comment]
I have figured out the issue, so anybody who has trouble with this kind of thing, just rename the current games file under steamapps\common to something else like [gamename]_backup, then create a new file and call it [gamename] and put all the files within depotdownloader into said file, then the game runs the older versions just like normal.
